css/partials/_layout.scss

```diff
@@ -263,4 +278,9 @@ html.no-js .js-hidden {
 	@include bp-tablet--portrait {
 		display: none;
 	}
+}
+
+.content {
+    margin-top: 36px;
+    clear: both;
```

Comment:
Is this comment from @frrrances something that needs to be adjusted in this branch? I don't see a subsequent commit that reacts to this comment?